src/stylesheets/elements/_links.scss

```diff
@@ -156,3 +156,77 @@ footer a  {
 [class*="hint"][class*="theme--"] a:link {
   text-decoration: none;
 }
+
+/*--- Search Global ---*/
+
+@include media($desktop) {
+  .search__global__toggle {
+    display: none;
+  }
+}
+
+@media (max-width: $desktop - 1) {
+  .search__global__toggle {
+    top: 0;
+    right: 0;
+    width: 60px;
```

Comment:
I thought fixed widths were frowned upon?

Reply:
Fixed heights yeh, not really widths. Plus this is an absolute within an absolute and also as each icon will have varying widths, its needs to be specified to match the size of the input-group icon.